cli/help.md

```diff
@@ -23,6 +23,7 @@ Basic options:
 --amd.autoId                Generate the AMD ID based off the chunk name
 --amd.basePath <prefix>     Path to prepend to auto generated AMD ID
 --amd.define <name>         Function to use in place of `define`
+--amd.keepExtension         Add `.js` extension for generated chunks and local AMD modules
```

Comment:
This file should wrap at 80 characters, can we slightly shorten this? E.g. "Add `.js` extension in imports"